docs/source/migration.mdx

```diff
@@ -616,6 +670,8 @@ Every GraphQL server supports a trivial query that requests the [`__typename`](/
 https://your.server/?query=%7B__typename%7D
 ```
 
+(Note that you should configure your health check to send a `apollo-require-preflight: true` request along with this method so that it is not blocked by the [CSRF prevention](/apollo-server/security/cors/#preventing-cross-site-request-forgery-csrf) security feature.)
```

Comment:
I think this would be a nice call out: 
```suggestion
> You should also send an `apollo-require-preflight: true` header alongside your health check, so it isn't blocked by the [CSRF prevention](/apollo-server/security/cors/#preventing-cross-site-request-forgery-csrf) feature.
```